Source/TurboLinkGrpc/Private/TurboLinkGrpcClient.cpp

```diff
@@ -37,6 +37,14 @@ EGrpcContextState UGrpcClient::GetContextState(FGrpcContextHandle Handle) const
 	return (*context)->GetState();
 }
 
+void UGrpcClient::AddMetadataToContext(FGrpcContextHandle Handle, FString Key, FString Value)
+{
+	const TSharedPtr<GrpcContext>* context = ContextMap.Find(Handle);
+	if (context == nullptr) return;
```

Comment:
nit: Not sure how blueprints usually do error handling, but I guess this is good in the spirit of "the show must go on".

Reply:
I can update this to return a bool depending on if it successfully adds the metadata, not exactly error handling but at least providing information to the caller if it was successful

Reply:
Honestly if the caller gets the ContextHandle from the same instance, they shouldn't need to do error handling. But the type design in this case makes it cleaner to at least return the bool IMO.